--- /home/agalat/stud/Kirill/thesis/thesis_tmp_thesis/synopsis.bbl	2025-10-10 08:16:26.713794430 +0300
+++ synopsis.bbl	2025-10-10 08:06:40.308637293 +0300
@@ -2691,14 +2691,13 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- PP. 369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О свойствах правильных семейств булевых функций}
       \field{year}{2021}
       \field{number}{1}
       \range{number}{1}
-      \field{pages}{91\bibrangedash 102}
+      \field{pages}{С.~91\bibrangedash 102. \\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- vol. 32, no. 5.~--- pp.~369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л}
       \range{pages}{12}
       \field{volume}{33}
       \range{volume}{1}
@@ -2733,14 +2732,15 @@
       \field{sortinit}{5}
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- PP. 203--217 \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О порождении $n$-квазигрупп с помощью правильных семейств функций}
       \field{year}{2023}
       \field{number}{1}
       \range{number}{1}
-      \field{pages}{35\bibrangedash 53}
+      \field{pages}{С.~35\bibrangedash 53. \\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л.\\
+Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы~1 и результаты раздела 6, 29 \%, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- vol. 35, no. 4.~--- pp.~203--217. \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л. \\
+Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы~1 и результаты раздела 6, 29 \%, объем 0.34~п.~л}
       \range{pages}{19}
       \field{volume}{35}
       \range{volume}{1}
@@ -2778,16 +2778,16 @@
       \field{sortinit}{5}
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: FUKEYM; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 0.98~п.~л., 25 \%, Царегородцеву К. Д. принадлежат разделы 4,5,6, объем 0.25~п.~л.}
       \field{journaltitle}{Математические вопросы криптографии}
-      \field{langid}{russian}
+      \field{langid}{english}
       \field{title}{Proper families of functions and their applications}
-      \field{year}{2023}
-      \field{number}{2}
+      \field{year}{2023.~--- T.~14, №~2}
+      %\field{number}{2}
       \range{number}{1}
-      \field{pages}{43\bibrangedash 58}
+      \field{pages}{С.~43\bibrangedash 58. \\ EDN: FUKEYM; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 0.98~п.~л.\\
+Царегородцеву К. Д. принадлежат разделы 4,5,6, 25 \%, объем 0.25~п.~л}
       \range{pages}{16}
-      \field{volume}{14}
+      %\field{volume}{14}
       \range{volume}{1}
       \keyw{recommended,biblioauthorvak,biblioauthor,bibliofull}
     \endentry
@@ -2810,14 +2810,13 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ Импакт-фактор: 0.117 (РИНЦ); 1.37~п.~л.}
       \field{journaltitle}{Интеллектуальные системы. Теория и приложения}
       \field{langid}{russian}
       \field{title}{Об индексе ассоциативности конечных квазигрупп}
       \field{year}{2024}
       \field{number}{3}
       \range{number}{1}
-      \field{pages}{80\bibrangedash 101}
+      \field{pages}{С.~80\bibrangedash 101. \\ Импакт-фактор: 0.117 (РИНЦ); 1.37~п.~л}
       \range{pages}{22}
       \field{volume}{28}
       \range{volume}{1}
@@ -2842,12 +2841,11 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: VTEBFJ; журнал индексируется в Scopus, RSCI. Импакт-фактор: 0.1 (JIF); общий объем 0.375~п.~л.}
       \field{journaltitle}{Прикладная дискретная математика}
       \field{langid}{russian}
       \field{title}{О взаимно однозначном соответствии между правильными семействами булевых функций и рёберными ориентациями булевых кубов}
       \field{year}{2020}
-      \field{pages}{16\bibrangedash 21}
+      \field{pages}{С.~16\bibrangedash 21. \\ EDN: VTEBFJ; журнал индексируется в Scopus, RSCI. Импакт-фактор: 0.1 (JIF); общий объем 0.375~п.~л}
       \range{pages}{6}
       \field{volume}{48}
       \range{volume}{1}
@@ -2872,14 +2870,13 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: EYLHYQ. Импакт-фактор: 0.117 (РИНЦ); общий объем 0.25~п.~л.}
       \field{journaltitle}{Интеллектуальные системы. Теория и приложения}
       \field{langid}{russian}
       \field{title}{О соответствии между правильными семействами и реберными ориентациями булевых кубов}
       \field{year}{2020}
       \field{number}{1}
       \range{number}{1}
-      \field{pages}{97\bibrangedash 100}
+      \field{pages}{С.~97\bibrangedash 100. \\ EDN: EYLHYQ. Импакт-фактор: 0.117 (РИНЦ); общий объем 0.25~п.~л}
       \range{pages}{4}
       \field{volume}{24}
       \range{volume}{1}
@@ -2913,16 +2910,16 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: ECXXNP; журнал индексируется в Scopus. Импакт-фактор: 0.28 (SJR); общий объем 0.81~п.~л., 38 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство результатов раздела 4, объем 0.31~п.~л.}
       \field{journaltitle}{Journal of Mathematical Sciences}
       \field{langid}{english}
       \field{title}{A Criterion of Properness for a Family of Functions}
-      \field{year}{2024}
-      \field{number}{4}
+      \field{year}{2024.~--- vol.~284, no.~4}
+      %\field{number}{4}
       \range{number}{1}
-      \field{pages}{451\bibrangedash 459}
+      \field{pages}{pp.~451\bibrangedash 459. \\ EDN: ECXXNP; журнал индексируется в Scopus. Импакт-фактор: 0.28 (SJR); общий объем 0.81~п.~л.\\
+Царегородцеву К. Д. принадлежат формулировка и доказательство результатов раздела 4, 38 \%, объем 0.31~п.~л}
       \range{pages}{9}
-      \field{volume}{284}
+      %\field{volume}{284}
       \range{volume}{1}
       \keyw{recommended,biblioauthorvak,biblioauthorscopus,biblioauthor,bibliofull}
     \endentry
@@ -2944,16 +2941,15 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: QMBWSF; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 1.31~п.~л.}
       \field{journaltitle}{Математические вопросы криптографии}
-      \field{langid}{russian}
+      \field{langid}{english}
       \field{title}{Format-preserving encryption: a survey}
-      \field{year}{2022}
-      \field{number}{2}
-      \range{number}{1}
-      \field{pages}{133\bibrangedash 153}
+      \field{year}{2022.~--- T.~13, №~2.}
+      %\field{number}{2}
+      %\range{number}{1}
+      \field{pages}{С.~133\bibrangedash 153. \\ EDN: QMBWSF; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 1.31~п.~л}
       \range{pages}{21}
-      \field{volume}{13}
+      %\field{volume}{13}
       \range{volume}{1}
       \keyw{recommended,biblioauthorvak,biblioauthor,bibliofull}
     \endentry
@@ -2976,12 +2972,11 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: NOTNOY. Импакт-фактор: 0.056 (РИНЦ); 0.15~п.~л.}
       \field{journaltitle}{Прикладная дискретная математика. Приложение}
       \field{langid}{russian}
       \field{title}{Об одном квазигрупповом алгоритме шифрования, сохраняющего формат}
       \field{year}{2023}
-      \field{pages}{102\bibrangedash 104}
+      \field{pages}{С.~102\bibrangedash 104. \\ EDN: NOTNOY. Импакт-фактор: 0.056 (РИНЦ); 0.15~п.~л}
       \range{pages}{3}
       \field{volume}{16}
       \range{volume}{1}
@@ -5661,14 +5656,13 @@
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- PP. 369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О свойствах правильных семейств булевых функций}
       \field{year}{2021}
       \field{number}{1}
       \range{number}{1}
-      \field{pages}{91\bibrangedash 102}
+      \field{pages}{91\bibrangedash 102. \\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- P.~369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л}
       \range{pages}{12}
       \field{volume}{33}
       \range{volume}{1}
@@ -5703,14 +5697,13 @@
       \field{sortinit}{5}
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- PP. 203--217 \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О порождении $n$-квазигрупп с помощью правильных семейств функций}
       \field{year}{2023}
       \field{number}{1}
       \range{number}{1}
-      \field{pages}{35\bibrangedash 53}
+      \field{pages}{35\bibrangedash 53. \\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- P.~203--217. \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л}
       \range{pages}{19}
       \field{volume}{35}
       \range{volume}{1}
@@ -5748,14 +5741,13 @@
       \field{sortinit}{5}
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: FUKEYM; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 0.98~п.~л., 25 \%, Царегородцеву К. Д. принадлежат разделы 4,5,6, объем 0.25~п.~л.}
       \field{journaltitle}{Математические вопросы криптографии}
-      \field{langid}{russian}
+      \field{langid}{english}
       \field{title}{Proper families of functions and their applications}
       \field{year}{2023}
       \field{number}{2}
       \range{number}{1}
-      \field{pages}{43\bibrangedash 58}
+      \field{pages}{43\bibrangedash 58. \\ EDN: FUKEYM; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 0.98~п.~л., 25 \%, Царегородцеву К. Д. принадлежат разделы 4,5,6, объем 0.25~п.~л}
       \range{pages}{16}
       \field{volume}{14}
       \range{volume}{1}
@@ -6045,7 +6037,7 @@
       \field{sortinithash}{ad6fe7482ffbd7b9f99c9e8b5dccd3d7}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- PP. 369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л.}
+      \field{addendum}{\\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- P.~369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О свойствах правильных семейств булевых функций}
@@ -6166,7 +6158,7 @@
       \field{sortinit}{6}
       \field{sortinithash}{b33bc299efb3c36abec520a4c896a66d}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- PP. 203--217 \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л.}
+      \field{addendum}{\\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- P.~203--217 \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О порождении $n$-квазигрупп с помощью правильных семейств функций}
@@ -6374,7 +6366,7 @@
       \field{sortinithash}{ad6fe7482ffbd7b9f99c9e8b5dccd3d7}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- PP. 369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л.}
+      \field{addendum}{\\ EDN: JTVVAY; журнал индексируется в RSCI. Импакт-фактор: 0.385 (РИНЦ); общий объем 0.75~п.~л.. \\ Перевод: \\ Tsaregorodtsev K.D. Properties of proper families of Boolean functions // Discrete Mathematics and Applications.~--- 2022.~--- Vol. 32, No. 5.~--- P.~369--378. \\ EDN: INXYMW; журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 0.75~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О свойствах правильных семейств булевых функций}
@@ -6449,7 +6441,7 @@
       \field{sortinit}{5}
       \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
       \field{labeltitlesource}{title}
-      \field{addendum}{\\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- PP. 203--217 \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л.}
+      \field{addendum}{\\ EDN: WWYSEG; журнал индексируется в RSCI Импакт-фактор: 0.385 (РИНЦ); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л. \\ Перевод: \\ Galatenko A.V., Nosov V.A., Pankratiev A.E., Tsaregorodtsev K.D. Generation of $n$-quasigroups by proper families of functions // Discrete Mathematics and Applications.~--- 2025.~--- Vol. 35, No. 4.~--- P.~203--217 \\ DOI: https://doi.org/10.4213/dm1749, журнал индексируется в WOS, Scopus. Импакт-фактор: 0.3 (JIF); общий объем 1.18~п.л., 29 \%, Царегородцеву К. Д. принадлежат формулировка и доказательство теоремы 1 и результаты раздела 6, объем 0.34~п.~л.}
       \field{journaltitle}{Дискретная математика}
       \field{langid}{russian}
       \field{title}{О порождении $n$-квазигрупп с помощью правильных семейств функций}
@@ -6497,7 +6489,7 @@
       \field{labeltitlesource}{title}
       \field{addendum}{\\ EDN: FUKEYM; журнал индексируется в RSCI. Импакт-фактор: 0.232 (РИНЦ); общий объем 0.98~п.~л., 25 \%, Царегородцеву К. Д. принадлежат разделы 4,5,6, объем 0.25~п.~л.}
       \field{journaltitle}{Математические вопросы криптографии}
-      \field{langid}{russian}
+      \field{langid}{english}
       \field{title}{Proper families of functions and their applications}
       \field{year}{2023}
       \true{nocite}
